picky-server/src/configuration.rs

```diff
@@ -15,6 +15,8 @@ const PICKY_INTERMEDIATE_CERT_ENV: &str = "PICKY_INTERMEDIATE_CERT";
 const PICKY_INTERMEDIATE_KEY_ENV: &str = "PICKY_INTERMEDIATE_KEY";
 const PICKY_SAVE_CERTIFICATE_ENV: &str = "PICKY_SAVE_CERTIFICATE";
 const PICKY_BACKEND_FILE_PATH_ENV: &str = "PICKY_BACKEND_FILE_PATH";
+const PICKY_DEN_PUBLIC_KEY_FILE_ENV: &str = "PICKY_DEN_PUBLIC_KEY_FILE";
```

Comment:
rename to "PICKY_PUBLIC_KEY_FILE" instead of "PICKY_DEN_PUBLIC_KEY_FILE", same for the private key